APIConsumer/APIConsumer.html

```diff
@@ -0,0 +1,32 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <script>
+        let requestForProducts = new XMLHttpRequest()
```

Comment:
1. Honestly, `XMLHttpRequest` is already a little bit redundant method and you should pay attention for [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch), it has simpler syntax and supported by all modern browsers.
2. For JS I strongly advise to put `;` after each row to make code style consistent.
3. `requestForProducts` is quite syntactically incorrect because it is unclear what is your variable responsible for: fetch, add, update products. You can just use `getProductsRequest`

Same cases for categories request